### PR TITLE
187 add sfn project description

### LINKED ISSUE #187: Adding Sing For Needs project description
We have quite a lot of details around code & design policies in our README, although we're missing the explanation of the project.

## Expected Behavior

Adding project explanation to the README sort of as a value proposition.

## Current Behavior

Currently it can be confusing to explain what the project is about, and it could easier convince people if they see at first glance what we try to reach here.

## Possible Solution

Answering the following questions might help the documentation flowing:
- What is sing for need? (elevator pitch)
- What problem trying to solve?
- Why sing for need could be a possible solution?
- How it works?

### PR DESCRIPTION
**Pull request by aonomike**
Fixes #187

### CODE REVIEW

**Review by javpet — COMMENTED** [review]
This is a really great start Mike! It's a really good generic overview, but to really "sell" the idea potentially we need to dig deeper. Try to think about it as, how would you sell this product to 3 different target groups: the artist, the charity champion and the donor.

My suggestion would be to maybe try to extend the description with the following questions as well (probably together with the product manager):
- Why this project came alive? Why this topic? What was the initial idea / need behind?
- What's the current struggle of artists, donors and charities have that this solution might solve?
- Who we want to talk to with this application? Who is our audience?
- How the platform will work as in a simplified steps?
- What's the roadmap for 2019?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## About this project
-:construction: UNDER CONSTRUCTION :construction:
+Sing for needs is a donation platform, which is meant to be a positive means for giving, inspired by music performances from artists, unknown and famous. The Artists get to see and choose the various causes to support through performances while getting a view of all the funds generated and the usage.
+
+The sing for needs project is currently under active developement by a team of volunteers under the [Agileventures](https://www.agileventures.org) platform, an official UK Charity (#1170963) dedicated to crowdsourced learning and project development. Under the umbrella of Championer one projects, [Sing for needs backed](https://github.com/agileVentures/sing_for_needs) uses [Elixir/Phoenix](https://phoenixframework.org/) framework, while the Front end uses [React framework](https://reactjs.org/) mainly to facilitate mentorship for the volunteers to learn these modern technologies.
```
Comment (yakryder) [inline README.md]: We should figure out how we want this capitalized and be consistent with what we choose -- `Sing for Needs`, `sing for needs`, `Sing for needs`. Mostly I have seen `Sing for Needs`. 

No matter what we use there is a definite advantage to having one way to capitalize that we use everywhere

**Review by yakryder — COMMENTED** [review]
_My suggestion would be to maybe try to extend the description with the following questions as well (probably together with the product manager):_
- _Why this project came alive? Why this topic? What was the initial idea / need behind?_
- _What's the current struggle of artists, donors and charities have that this solution might solve?_
- _Who we want to talk to with this application? Who is our audience?_
- _How the platform will work as in a simplified steps?_
- _What's the roadmap for 2019?_

^^ I agree with Peter. It looks good and these are also good questions to ask. We should all spend a little time thinking about questions like these and -- in a perfect world where engineers know the product very well and product owners are very technical -- would all be able to answer them well.  

I also think there's an argument to be made for getting it in and iterating. It is a big improvement from the _About_ section's current state

File: README.md (hunk 1 above)
Comment (aonomike) [reply README.md]: Thanks @SherSpock maybe an iteration is key.

Comment (FedericoEsparza) [reply README.md]: Let's go with **_Sing for Needs_** for marketing and outward-facing purposes.

**Comment by FedericoEsparza** [comment]
@javpet I think those are good questions that we should address, but they might fit better in the actual About page in our app, rather than clutter the README with those answers. 

The PR addresses all the issue's concerns concisely.

I'm in favor of merging this PR, and creating a new ticket, where we notify @RachitKansal , our Product Manager, who's working on the mockups, so that he makes room for them.

I'll then answer the questions in the new ticket, so that they're ready to go for implementing the About mockup.

**Comment by javpet** [comment]
Yep @FedericoEsparza I think the purpose of my comment was more putting out there, so we have documented somewhere, I also felt it might be too much to tackle with this PR.

**Comment by FedericoEsparza** [comment]
@aonomike Can you standardize the capitalization for **_Sing for Needs_**, in your description, please.

And can you modify this line to be:
```
Under the umbrella of Championer projects, the [Sing for Needs backend]
```
And `frontend` should be one word, to mirror our use of `backend`.

Let's merge it in, after that.

**Review by FedericoEsparza — APPROVED** [review]
@aonomike Thanks for getting this done, Mike.

**Pull request merged**